assets/src/components/amp-support/validation-results-notice.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AMPNotice,
+	NOTICE_SIZE_SMALL,
+	NOTICE_TYPE_INFO,
+	NOTICE_TYPE_SUCCESS,
+	NOTICE_TYPE_WARNING,
+} from '../amp-notice';
+
+/**
+ * Render notices on site support page.
+ *
+ * @param {Object} props                       Component props.
+ * @param {Object} props.data                  Support data.
+ * @param {Object} props.args                  Support argument.
+ * @param {Object} props.ampValidatedPostCount AMP validated post counts.
+ * @return {JSX.Element|null} HTML markup for notice data.
+ */
+export function ValidationResultsNotice( { data, args, ampValidatedPostCount } ) {
+	const isSpecificUrlRequest = 0 < args?.urls?.length;
+	const hasErrors = 0 < data.errors?.length || 0 < data.urls?.length;
+	const hasAllStaleResult = 0 < ampValidatedPostCount.all && ampValidatedPostCount.all === ampValidatedPostCount.stale;
+	const hasSomeStaleResult = 0 < ampValidatedPostCount.fresh && 0 < ampValidatedPostCount.stale;
+	const hasAllFreshResult = 0 < ampValidatedPostCount.all && ampValidatedPostCount.all === ampValidatedPostCount.fresh;
```

Comment:
NIT: How about using plural nouns in the constant names, i.e. `hasAllStaleResults`, `hasSomeStaleResults` and `hasAllFreshResult`?

Reply:
I have addressed this in [c98d9be](https://github.com/ampproject/amp-wp/pull/6718/commits/c98d9be81a02c762c1950e2da6876592b91bf3af)